### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by Lioness100**
### Description of the changes

This PR fixes a few random typos (this was a trial run for [a CLI I made](https://github.com/Lioness100/rspell))
I think "bumpped_shuffle" is a typo, but it was used in a lot of places in the code, so I didn't want to change anything without clearing it with a maintainer.

Also, I know this edits both code and comments, and modifies a lot of different sections of the code, so let me know if you need me to split it up into more modular PRs. :)

### Have the changes in this PR been tested?

No

### CODE REVIEW

File: redbot/cogs/trivia/trivia.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ async def triviaset_allowoverride(self, ctx: commands.Context, enabled: bool):
             )
 
     @triviaset.command(name="usespoilers", usage="<true_or_false>")
-    async def trivaset_use_spoilers(self, ctx: commands.Context, enabled: bool):
```
Comment (Flame442) [inline redbot/cogs/trivia/trivia.py:157]: All changes in this file are inaccurate, as the command name is the literal `triviaset`, standing for "trivia setting".

File: redbot/cogs/mutes/mutes.py [hunk 2]
```diff
@@ -428,7 +428,7 @@ async def _auto_channel_unmute_user_multi(
         for result in results:
             if not result:
                 continue
-            _mmeber, channel, reason = result
+            _mmember, channel, reason = result
```
Comment (Flame442) [inline redbot/cogs/mutes/mutes.py]:
```suggestion
            _member, channel, reason = result
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
Much more accurate than most other automated spellchecking tools that have made PRs to Red, I'll give you that!

File: redbot/cogs/trivia/trivia.py (hunk 1 above)
Comment (Lioness100) [reply redbot/cogs/trivia/trivia.py:157]: I'm sorry, I think my intent was correct but I didn't fix it correctly. The typo was that it's "trivaset" (missing the "i" after the "v"). The problem was that I only fixed the trivia typo and forgot to add "set" with it.

Comment (Flame442) [reply redbot/cogs/trivia/trivia.py:157]: Ah, yep I see that now. If you want to make that change but keep `set` in the command name, that would be fine!

**Review by Flame442 — APPROVED** [review]
Thanks!

**Comment by Lioness100** [comment]
Np! Just as a note, do you know if "bumpped_shuffle" is a typo?

**Pull request merged**